manual/src/main/asciidoc/whats-new.adoc

```diff
@@ -51,26 +51,85 @@ More details about JSON Schemas implementation are available in the <<JSON schem
 
 The introduction of JSON schema required us to modify Apache Unomi data model, One of the key differences is the removal of open maps.
 
-For example, the following event model in Apache Unomi 1.x:
+The properties field in the events objects provided by unomi are now restricted by JSON schema.
+It's not possible to add value in this property without adding the JSON schema which allows to add the properties.
```

Comment:
```
...now restricted by JSON schema. This means object properties must be declared in a JSON schema for an event to be accapted.
```

Instead of `It's not possible[...]add the properties.`